### PR TITLE
Refactor processing scripts

### PR DESCRIPTION
**Pull request by suiguoxin**
- change default montage type to `bipolar`
- optimize memory utilization by avoid using a large dict of DataFrame
- code refactor

### CODE REVIEW

File: docs/configs/transformer_tusz_v1.5.2.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 data:
   type: detection
-  data_folder: data/tusz_processed_v1.5.2/unipolar/
-  meta_path: data/tusz_processed_v1.5.2/unipolar/meta.pkl
+  data_folder: data/tusz_processed_v1.5.2/bipolar/
+  meta_path: data/tusz_processed_v1.5.2/bipolar/meta.pkl
   num_variables: 19
```
Comment (chenyuqi990215) [inline docs/configs/transformer_tusz_v1.5.2.yml:5]: what is the purpose of changing to bipolar?

Comment (suiguoxin) [reply docs/configs/transformer_tusz_v1.5.2.yml:5]: To be consistent with official TUH documentation, use the montage they recommend.

Comment (chenyuqi990215) [reply docs/configs/transformer_tusz_v1.5.2.yml:5]: fine.

Comment (chenyuqi990215) [reply docs/configs/transformer_tusz_v1.5.2.yml:5]: please make sure num_variables is correct.

File: docs/configs/transformer_tusz_v2.0.0.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 data:
   type: detection
-  data_folder: data/tusz_processed_v2.0.0/unipolar/
-  meta_path: data/tusz_processed_v2.0.0/unipolar/meta.pkl
+  data_folder: data/tusz_processed_v2.0.0/bipolar/
+  meta_path: data/tusz_processed_v2.0.0/bipolar/meta.pkl
   num_variables: 19
```
Comment (chenyuqi990215) [inline docs/configs/transformer_tusz_v2.0.0.yml:5]: make sure here again.

File: docs/tusz.md [hunk 3]
```diff
@@ -17,6 +17,7 @@ Run `python scripts/tusz.py -h` to see help messages.
 # v1.5.2
 python scripts/tusz.py \
     --tusz-version 1.5.2 \
+    --montage-type 'bipolar' \
```
Comment (chenyuqi990215) [inline docs/tusz.md:20]: is '' redundant here?

File: docs/tusz.md [hunk 4]
```diff
@@ -29,11 +30,12 @@ python scripts/tusz.py \
 # original frequecies: [250, 256, 400]
 python scripts/tusz.py \
     --tusz-version 2.0.0 \
+    --montage-type 'bipolar' \
```
Comment (chenyuqi990215) [inline docs/tusz.md:33]: also here?

File: docs/configs/transformer_tusz_v2.0.0.yml (hunk 2 above)
Comment (suiguoxin) [reply docs/configs/transformer_tusz_v2.0.0.yml]: Fixed

File: docs/tusz.md (hunk 3 above)
Comment (suiguoxin) [reply docs/tusz.md]: Removed

**Pull request merged**